frontend/src/lib/pages/NnsAccounts.svelte

```
@@ -43,14 +43,15 @@
   };
 
   const handleAction = ({ detail }: { detail: Action }) => {
+    const account = findAccount({
+      identifier: detail.data.accountIdentifier,
+      accounts: $nnsAccountsListStore,
+    });
+    if (isNullish(account)) {
+      return;
```

This shouldn't happen, right?
Should we throw an error, or log to the console or something?

---

Correct, this shouldn't happen. Yes, I can add a toast error.

---

done